tensorflow_serving/model_servers/http_server.cc

```diff
@@ -176,7 +176,7 @@ class RestApiRequestDispatcher {
     req->ReplyWithStatus(http_status);
 
     RecordRequestLatency(model_name, /*api=*/method, /*entrypoint=*/"REST",
-                         Env::Default()->NowMicros() - start);
+                         status, Env::Default()->NowMicros() - start);
```

Comment:
maybe i should have clarified. i think we only need to track of latency of **successful** requests. for errored out requests, i do not see it being relevant. you want to remove "status" as a dimension and only increment/record latency for successful requests at call site.